.circleci/config.yml

```diff
@@ -94,16 +128,26 @@ jobs:
                 echo "Skipping integration tests"
             fi
           no_output_timeout: 3000s
-      - run:
-          name: Combine Coverage Results
-          working_directory: ~/curation
-          command: |
-              . tests/combine_coverage.sh
-          no_output_timeout: 30s
-      # Save test results
-      - store_test_results:
-          path: ~/curation/tests/results/junit
-      # Save artifacts
-      - store_artifacts:
-          path: ~/curation/tests/results/coverage
-          destination: test_results
+      - test_teardown
+
+workflows:
+  version: 2
+  test:
+    jobs:
+      - unit_test
+      - integration_test:
+          filters:
+            branches:
+              # This would cause too much noise to run on commit, run nightly
+              # instead per below workflow.
+              ignore: develop
```

Comment:
I take this to mean, integration tests won't run on develop when a PR is merged, but will run nightly on develop.  Correct?

Reply:
Yep. When a PR is merged, just the unit tests will run.